presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConnectorFactory.java

```diff
@@ -83,7 +83,7 @@ public String getName()
     @Override
     public ConnectorHandleResolver getHandleResolver()
     {
-        return new HiveHandleResolver();
+        return null;
```

Comment:
Maybe add a comment
```java
// we use a separate handle resolver per connector instance
return null;
```
Or we can delete this method if we make it a default interface method.